src/web/web-service/frontend/res/js/app.js

```diff
@@ -372,5 +381,403 @@
                 ev.target.disabled = false;
             });
         };
+
+        $scope.restServerPort = '8080';
```

Comment:
is 8080 the default port of otbr-web?

Reply:
On my side,  80 port in docker is mapped to 8080 which is the port for rest API, and otbr-web use 81 port in docker which is mapped to 8081.